docs/layouts/shortcodes/generated/execution_config_configuration.html

```diff
@@ -58,6 +58,12 @@
             <td><p>Enum</p></td>
             <td>The NOT NULL column constraint on a table enforces that null values can't be inserted into the table. Flink supports 'error' (default) and 'drop' enforcement behavior. By default, Flink will check values and throw runtime exception when null values writing into NOT NULL columns. Users can change the behavior to 'drop' to silently drop such records without throwing exception.<br /><br />Possible values:<ul><li>"ERROR"</li><li>"DROP"</li></ul></td>
         </tr>
+        <tr>
+            <td><h5>table.exec.sink.pk-shuffle</h5><br> <span class="label label-primary">Streaming</span></td>
```

Comment:
Can we create separate PR for this?

Reply:
Okay, I thought the target issue is the same one. I'll create a separate pr.